executor/prepared.go

```diff
@@ -236,11 +225,6 @@ type DeallocateExec struct {
 	Name string
 }
 
-// Next implements the Executor Next interface.
-func (e *DeallocateExec) Next(ctx context.Context) (Row, error) {
-	return nil, errors.Trace(e.run(ctx))
```

Comment:
Put the code of e.run here.

Reply:
why?

Reply:
`e.run` is only called by `NextChunk`, we no longer need it.

Reply:
The logic of this code is very simple,
no need to be extracted since Next is removed.

Reply:
`e.run` can also be removed.

Reply:
@yangwenmai plz address the comment of @XuHuaiyu